pkg/function/kubeops.go

```diff
@@ -67,7 +66,7 @@ func (crs *kubeops) Exec(ctx context.Context, tp param.TemplateParams, args map[
 	if err := Arg(args, KubeOpsOperationArg, &op); err != nil {
 		return nil, err
 	}
-	if err := OptArg(args, KubeOpsNamespaceArg, &namespace, metav1.NamespaceDefault); err != nil {
+	if err := OptArg(args, KubeOpsNamespaceArg, &namespace, nil); err != nil {
```

Comment:
passing `default` namespace is a bit confusing for cluster scoped resources. I think empty string would also work. I will let Prasad approve.

Reply:
The safest way I can think of is having an additional arg through which we can decide if the resource is cluster scoped. 

Another approach I can think of is running discovery and checking if the passed resource is cluster scoped and ignoring namespace in that case.

Reply:
In k8s cluster scoped resources are created simply without specifying a namespace. There is no explicit argument in the yaml. Handling this different in Kanister feels a bit confusing.

Latter approach should throw an error when specifying an namespace on a cluster scoped resource. Otherwise it would be quite confusing cause of the magic happening in the background.